Extensions/mutualchecker.js

```diff
@@ -86,12 +86,33 @@ XKit.extensions.mutualchecker = new Object({
 	check: function(json_obj, $link) {
 		if (json_obj.following && !json_obj.customizable) {
 			if (typeof this.mutuals[json_obj.name] === "undefined") {
-				XKit.interface.is_following(json_obj.name, this.preferences.main_blog.value).then(function(follow) {
-					if (follow) {
-						XKit.extensions.mutualchecker.add_label($link, json_obj.name);
-						XKit.extensions.mutualchecker.mutuals[json_obj.name] = true;
-					} else {
-						XKit.extensions.mutualchecker.mutuals[json_obj.name] = false;
+				XKit.tools.Nx_XHR({
+					method: "GET",
+					url: "https://www.tumblr.com/svc/conversations/participant_info?q=" + json_obj.name + "&participant=" + this.preferences.main_blog.value,
```

Comment:
do you write `https://www.tumblr.com/` for every one of these URLs, or is that something you could bake into your `XKit.tools.Nx_XHR()` helper to reduce the redundancy?